code/modules/mob/mob_lists.dm

```diff
@@ -121,7 +121,14 @@
 /mob/proc/add_to_current_living_antags()
 	if(!SSticker?.mode)
 		return
-	SSticker.mode.current_players[CURRENT_LIVING_ANTAGS] |= src
+
+	if (length(mind.antag_datums) == 0)
```

Comment:
if !length also works here and is arguably better

Reply:
Should we change the entire file then? They all have the same convention as this